src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -84,35 +106,42 @@ public OSIsoftServer() {}
    * @param Headers Request headers
    * @param TextFields Request Payload
    * @param FileName File name for response
-   * @return The HTTP response code
+   * @return True/False message was successfully sent
    */
   public static boolean RequestHTTPS(

```

Comment:
```suggestion
  public static boolean RequestHttps(
```

Google Java Style Guide does not allow for acronyms to be capitalized. See https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

While this PR did not directly address the name of this method, we should either add a commit to address it or create a GitHub issue to fix later.

Reply:
To add some context, I think the rule was previously that acronyms of only two characters were permitted, but that has since changed. 

You may notice some library or other classes using two-letter capitalized acronyms in variable or method names as previously permitted by Google Code Style.